Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/util/UiInteractions.java

```diff
@@ -108,7 +112,13 @@ public String getDescription() {
 
 			@Override
 			public void perform(UiController uiController, View view) {
-				((SeekBar) view).setProgress(progress);
+				try {
+					Method privateSetProgressMethod = ProgressBar.class.getDeclaredMethod("setProgressInternal", Integer.TYPE, Boolean.TYPE, Boolean.TYPE);
+					privateSetProgressMethod.setAccessible(true);
+					privateSetProgressMethod.invoke(view, progress, true, true);
+				} catch (ReflectiveOperationException e) {
```

Comment:
If this is a helper method for a test we should throw the exception so the test fails

Reply:
The problem is, that this is an overwritten method from a java class, which doesn't throw the exception. As the ReflectiveOperationException is a checked exception we can't just re-throw it.